compiler/rustc_session/src/config.rs

```diff
@@ -1762,6 +1762,12 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
         debugging_opts.symbol_mangling_version = SymbolManglingVersion::V0;
     }
 
+    if let Some(env) = std::env::var_os("RUSTC_GRAPHVIZ_FONT") {
+        if let Ok(graphviz_font) = env.into_string() {
+            debugging_opts.graphviz_font = graphviz_font;
```

Comment:
FWIW, this is equivalent to `env::var` since you continue silently when there's non-UTF data. This super doesn't matter here, and there's no need to change it, but maybe it helps you in the future.

Reply:
lol, yeah, that makes sense now. Good to know for the future. Thanks!

Reply:
Oh, since you delegated this, I'm going to make the change.

Reply:
I'll want for checks to compile, and also test locally one more time with and without the env setting.

Then I'll push it.

Reply:
(Verified `.dot` file generates the expected font with and without the environment variable set... Using `env::var` now.)